### PR TITLE
Update and rename xchat.xml to hexchat.xml

### PR DESCRIPTION
**Pull request by theatre-x**
updated name

### CODE REVIEW

**Comment by Mailaender** [comment]
Might want to keep both. XChat may be dead, but in Linux distribution repositories it is still available.

**Comment by ROCKNROLLKID** [comment]
I think we can merge this. He put in the description that it was formerly called XChat. 

Besides, a lot of the old Linux distros will be dropped when Python 3/GTK 3 will be done. New min version will be Ubuntu 14.04

File: cleaners/hexchat.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <!--
 
     BleachBit
-    Copyright (C) 2014 Andrew Ziem
+    Copyright (C) 2015 Andrew Ziem
```
Comment (az0) [inline cleaners/hexchat.xml]: This line is causing a merge conflict

**Comment by az0** [comment]
This is just a name change, and it can still clean XChat files, so merging this is OK. The issue is the merge conflict.

**Comment by ROCKNROLLKID** [comment]
I don't think theatre-x is around anymore, at least I have not seen him active in the BleachBit community in over a year. I can go ahead and open a new PR for this. Is the year suppose to be 2016 or 2008-2016?

**Comment by az0** [comment]
The year can be just left off the change. I'll close this then.

**Pull request closed without merging**